.github/workflows/byob-gradle-trw.yml

```diff
@@ -0,0 +1,78 @@
+# Copyright 2023 SLSA Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+
+on:
+  workflow_call:
+    inputs:
+      name1:
```

Comment:
do we need to rename input `name1` and `name2`?

Reply:
Removed since the Gradle builder doesn't need them, and I believe the BYOB-side doesn't either.